src/controllers/movies.js

```diff
@@ -0,0 +1,182 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+// const createCustomer = async (req, res) => {
+//     const {
+//         name,
+//         phone,
+//         email
+//     } = req.body
+
```

Comment:
This is considered "dead code" (code that is commented out), it should be removed before submitting a PR in most cases